h2o-core/src/main/java/water/Keyed.java

```diff
@@ -57,6 +58,13 @@ public static void remove( Key k ) {
     if (val==null) return;
     ((Keyed)val.get()).remove();
   }
+  public static void removeQuietly(Key k) {
+    try {
+      remove(k);
+    } catch (Exception e) {
+      Log.trace("Failed to remove key " + k, e);
```

Comment:
log level looks a bit too low: what about `warn` for the failure+error message, and `debug` for the stacktrace?

Reply:
fixed